index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="./scss/main.css">
+  <title>WTV - Wine Tasting (out in the) Vineyard</title>
+</head>
+
+<body>
+  <header class="header">
+
+    <!-- top bar: hidden for mobile -->
+    <div class="topBar flex-jc-end">
+      <a href="#">
+        <img class="socials" src="img/facebook.svg" alt="facebook logo icon" />
+      </a>
+      <a href="#">
+        <img class="socials" src="img/twitter.svg" alt="twitter logo icon" />
+      </a>
+      <a href="#">English</a>
+      <a href="#">My Page</a>
+      <a href="#">Logout</a>
+    </div>
+
```

Comment:
- [ ] Please make use of the `article` tag for the articles section as part of the semantic HTML implementation.

Reply:
The required change is done. Although, I am not sure if the use of the `article` tag for a  header top bar is a correct implementation of semantic HTML.